js/aax.js

```diff
@@ -98,7 +98,7 @@ module.exports = class aax extends Exchange {
                 'fetchTradingFees': false,
                 'fetchTradingLimits': undefined,
                 'fetchTransactionFee': undefined,
-                'fetchTransactionFees': undefined,
+                'fetchTransactionFees': true,
```

Comment:
If you can 
- pass a code to the exchange, or (which you can by passing the currency argument to the [api endpoint](https://www.aax.com/apidoc/index.html#get-currencies))
- `fetchTransactionFees` returns the results of every available currency

then `fetchTransactionFee` should be `'emulated'` because it is [implemented on the super class using `fetchTransactionFees` ](https://github.com/ccxt/ccxt/blob/7d016471ba3f6d0b8f1ba347ddcbf487234a12ac/js/base/Exchange.js#L2052)

Reply:
Related to the currency argument, there are a couple of duplicate currencies because each network is a different object.
For example - USDT:
```
[
    {
        "chain": "ERC20USDT",
        "displayName": "TetherUS",
        "withdrawFee": "4",
        "withdrawMax": "0",
        "withdrawMin": "20",
        "otcFee": "0",
        "enableOTC": true,
        "visible": true,
        "enableTransfer": true,
        "transferMin": "2",
        "depositMin": "1",
        "enableWithdraw": true,
        "enableDeposit": true,
        "isLighting": false,
        "addrWithMemo": false,
        "addrSupportFIO": true,
        "withdrawPrecision": "0.000001",
        "currency": "USDT",
        "network": "ERC20",
        "minConfirm": "12"
    },
    {
        "chain": "TRC20USDT",
        "displayName": "TetherUS",
        "withdrawFee": "1",
        "withdrawMin": "20",
        "otcFee": "0",
        "enableOTC": true,
        "visible": true,
        "enableTransfer": true,
        "transferMin": "2",
        "depositMin": "1",
        "enableWithdraw": true,
        "enableDeposit": true,
        "isLighting": false,
        "addrWithMemo": false,
        "addrSupportFIO": true,
        "withdrawPrecision": "0.000001",
        "currency": "USDT",
        "network": "TRC20",
        "minConfirm": "20"
    }
]
```

How do we handle this @samgermain @carlosmiei? I assume the best way to address this is by not using the currency param in the request such that we can return the withdraw fees (in the case of aax) for all the different networks?
```js
   'USDT': {
       'withdraw': {
           'ERC20': 4,
           'TRC20': 1,
       },
       'deposit': {},
       'info': { ... },
   }
```